px4_ros2_cpp/include/px4_ros2/control/setpoint_types/experimental/trajectory.hpp

```diff
@@ -36,6 +36,9 @@ class TrajectorySetpointType : public SetpointBase
     std::optional<float> yaw_ned_rad = {},
     std::optional<float> yaw_rate_ned_rad_s = {});
 
+  void updatePosition(
```

Comment:
For position only, there's already a goto setpoint: https://github.com/Auterion/px4-ros2-interface-lib/blob/main/px4_ros2_cpp/include/px4_ros2/control/setpoint_types/goto.hpp
Does that not fit your use-case?

Reply:
The go-to setpoint did not work as well. I am also controlling velocity in my application.

Reply:
Ok I don't mind adding this, but please add a comment to generally prefer the goto setpoint. It's also possible to switch between multiple setpoint types in the same mode.

There's also some CI failures (you can rebase for the build failure).